plugins/org.jboss.tools.jst.web.ui/src/org/jboss/tools/jst/web/ui/WebUiPlugin.java

```diff
@@ -264,6 +264,14 @@ public static JSPTextEditor getActiveEditor() {
 		return editor;
 	}
 
+	public static JSPMultiPageEditor getActiveMultiPageEditor() {

```

Comment:
*must* this be our multipageeditor that is returned ? 

then there is a twoway depeendency on how this works is there not?